src/vscode-dts/vscode.proposed.interactive.d.ts

```diff
@@ -9,6 +9,13 @@ declare module 'vscode' {
 		command: string;
 		detail?: string;
 		refer?: boolean;
+		/**
+		 * Whether the command accepts arguments.
+		 * Defaults to `true`. When set to `false`,
+		 * accepting this command from the suggest
+		 * widget will also immediately execute it.
+		 */
+		hasArgs?: boolean;
```

Comment:
I would probably just call it `executeImmediately` or something like that. That would be more clear to me and let the undefined value be the same as the false value